.devcontainer/docker-compose.yml

```
@@ -0,0 +1,40 @@
+    restart: unless-stopped
+    environment:
+      - KEYCLOAK_ADMIN=${KEYCLOAK_ADMIN}
+      - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD}
+      - KC_DB=dev-file
+      - KC_HTTP_ENABLED=true
+      - KC_HOSTNAME_STRICT=false
+      - KC_HOSTNAME_STRICT_HTTPS=false
+    network_mode: service:couchdb
+    env_file:
+      - .env
+    command: start-dev --http-port=8090
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)